synapse/storage/data_stores/main/schema/delta/58/cache_instance.sql.postgres

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* for some reason, we have accumulated duplicate entries in
+  * device_lists_outbound_pokes, which makes prune_outbound_device_list_pokes less
+  * efficient.
+  */
+
+DROP TABLE IF EXISTS cache_invalidation_stream;
+
+CREATE TABLE cache_invalidation_stream (
```

Comment:
I don't know if we care, but re-using this table name means it will be hard to downgrade.

Reply:
Good point, I've renamed it and truncated the old table.